controller/config.go

```diff
@@ -136,28 +144,35 @@ const (
 	DefaultMaxTxnLogCollectionMB = 10 // 10 MB
 )
 
-// ControllerOnlyConfigAttributes are attributes which are only relevant
-// for a controller, never a model.
-var ControllerOnlyConfigAttributes = []string{
-	AllowModelAccessKey,
-	APIPort,
-	AutocertDNSNameKey,
-	AutocertURLKey,
-	CACertKey,
-	ControllerUUIDKey,
-	IdentityPublicKey,
-	IdentityURL,
-	SetNUMAControlPolicyKey,
-	StatePort,
-	MongoMemoryProfile,
-	MaxLogsSize,
-	MaxLogsAge,
-	MaxTxnLogSize,
-	AuditingEnabled,
-	AuditLogCaptureArgs,
-	AuditLogMaxSize,
-	AuditLogMaxBackups,
-}
+var (
+	// ControllerOnlyConfigAttributes are attributes which are only relevant
+	// for a controller, never a model.
+	ControllerOnlyConfigAttributes = []string{
+		AllowModelAccessKey,
+		APIPort,
+		AutocertDNSNameKey,
+		AutocertURLKey,
+		CACertKey,
+		ControllerUUIDKey,
+		IdentityPublicKey,
+		IdentityURL,
+		SetNUMAControlPolicyKey,
+		StatePort,
+		MongoMemoryProfile,
+		MaxLogsSize,
+		MaxLogsAge,
+		MaxTxnLogSize,
+		AuditingEnabled,
+		AuditLogCaptureArgs,
+		AuditLogMaxSize,
+		AuditLogMaxBackups,
+		AuditLogExcludeMethods,
+	}
+
+	// DefaultAuditLogExcludeMethods is the default list of methods to
+	// exclude from the audit log - just the one for `juju status`.
+	DefaultAuditLogExcludeMethods = []string{"Client.FullStatus"}
```

Comment:
Why not most/all other read-only methods by default?

Reply:
After some discussion with @jameinel I agreed with him that for auditihng it's better to err on the side of filtering less by default ([discussion](https://github.com/juju/juju/pull/8242#issuecomment-352670932)), especially once it's configurable and can be changed in a running controller. If an operator wants more filtered, they can add them pretty easily.